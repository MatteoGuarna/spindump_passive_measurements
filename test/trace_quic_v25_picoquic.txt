Draft-25 test with Picoquic, base setup, 0-RTT, quantum readiness, and key update

