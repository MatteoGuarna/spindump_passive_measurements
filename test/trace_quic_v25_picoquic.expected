QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 0866243c54e88ae9-ed3f867ca0cb93b4 (57700:4433) at 1580747823793171 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 0866243c54e88ae9-08bcb0b8251ea698 (57700:4433) at 1580747823892385 change starting packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 0866243c54e88ae9-08bcb0b8251ea698 (57700:4433) at 1580747823892385 measurement up right 99214 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 66243c54e88ae9e8-08bcb0b8251ea698 (57700:4433) at 1580747823892395 packet up responder length 1280 packets 1 2 bytes 1280 2560 bandwidth 1280 2560 note "V25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 66243c54e88ae9e8-08bcb0b8251ea698 (57700:4433) at 1580747823892804 packet up responder length 1261 packets 1 3 bytes 1280 3821 bandwidth 1280 3821 note "V25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 66243c54e88ae9e8-08bcb0b8251ea698 (57700:4433) at 1580747823893771 packet up initiator length 1280 packets 2 4 bytes 2560 3951 bandwidth 2560 3951 note "V25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 66243c54e88ae9e8-08bcb0b8251ea698 (57700:4433) at 1580747823992051 measurement up left 582 packets 5 9 bytes 4266 9382 bandwidth 4266 9382 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 66243c54e88ae9e8-08bcb0b8251ea698 (57700:4433) at 1580747824089163 measurement up right 97112 packets 8 10 bytes 4558 10868 bandwidth 4558 10868 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 66243c54e88ae9e8-08bcb0b8251ea698 (57700:4433) at 1580747824089163 measurement up full (responder) 97694 packets 8 10 bytes 4558 10868 bandwidth 4558 10868 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08e13e9d2618a267-4c31948576ee5756 (57702:4433) at 1580747829301326 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08e13e9d2618a267-0869364133d14606 (57702:4433) at 1580747829399767 change starting packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08e13e9d2618a267-0869364133d14606 (57702:4433) at 1580747829399767 measurement up right 98441 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f e13e9d2618a26728-0869364133d14606 (57702:4433) at 1580747829400555 packet up initiator length 1280 packets 2 4 bytes 2560 5120 bandwidth 2560 5120 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08ce29827de4edff-7902a748da477152 (50172:4433) at 1580747900326834 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08ce29827de4edff-7902a748da477152 (50172:4433) at 1580747900327755 packet starting initiator length 1280 packets 2 0 bytes 2560 0 bandwidth 2560 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08ce29827de4edff-08fbdfeac8a29445 (50172:4433) at 1580747900424236 change starting packets 2 0 bytes 2560 0 bandwidth 2560 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f 08ce29827de4edff-08fbdfeac8a29445 (50172:4433) at 1580747900424236 measurement up right 96481 packets 2 0 bytes 2560 0 bandwidth 2560 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900424372 packet up responder length 1280 packets 2 2 bytes 2560 1379 bandwidth 2560 1379 note "V25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900425345 packet up initiator length 1280 packets 3 13 bytes 3840 15459 bandwidth 3840 15459 note "V25,no spin"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900522508 measurement up left 370 packets 7 16 bytes 5649 18122 bandwidth 5649 18122 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900619227 measurement up right 96719 packets 10 20 bytes 5958 22065 bandwidth 5958 22065 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900619227 measurement up full (responder) 97089 packets 10 20 bytes 5958 22065 bandwidth 5958 22065 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900619786 measurement up left 559 packets 10 21 bytes 5958 22726 bandwidth 5958 22726 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900619786 measurement up full (initiator) 97278 packets 10 21 bytes 5958 22726 bandwidth 5958 22726 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900716544 measurement up right 96758 packets 12 22 bytes 6121 24214 bandwidth 6121 24214 note "V25,spinning"
QUIC 2a00:79e1:abc:301:2d7d:a1cc:d121:c516 <-> 2600:1f18:2310:d230:5103:7d9e:7d75:374f ce29827de4edff6c-08fbdfeac8a29445 (50172:4433) at 1580747900716544 measurement up full (responder) 97317 packets 12 22 bytes 6121 24214 bandwidth 6121 24214 note "V25,spinning"
